packages/cordova-plugin/src/web.ts

```diff
@@ -1,25 +1,54 @@
 import { require } from "cordova";
-import { ClearWatchOptions, PluginError, Position, PositionOptions } from "./definitions";
-import { ClearWatchOptionsDefault, PositionOptionsDefault } from "./defaults";
+import { ClearWatchOptions, OSGLOCPosition, PluginError, Position, CurrentPositionOptions, WatchPositionOptions } from "./definitions";
+import { ClearWatchOptionsDefault, CurrentPositionOptionsDefault, WatchPositionOptionsDefault } from "./defaults";
 
 var exec = require('cordova/exec');
 
-function getCurrentPosition(options: PositionOptions, success: (output: Position) => void, error: (error: PluginError) => void): void {
-  options = options || PositionOptionsDefault;
-
-  exec(success, error, 'OSGeolocation', 'getCurrentPosition', [options]);
+function getCurrentPosition(options: CurrentPositionOptions, success: (output: Position) => void, error: (error: PluginError) => void): void {
+  options = { ...CurrentPositionOptionsDefault, ...options };
+
+  let convertOnSuccess = (position: OSGLOCPosition) => {
+    let convertedPosition: Position = {
+      coords: {
+        latitude: position.latitude,
+        longitude: position.longitude,
+        altitude: position.altitude,
+        accuracy: position.accuracy,
+        heading: position.heading,
+        speed: position.speed,
+        altitudeAccuracy: position.accuracy
```

Comment:
Should this be `position.altitudeAccuracy`?

Reply:
Yes, nice catch! Fixed! Can you check?